### ISSUE TITLE
allow running buildifier from other directories

### ISSUE THREAD

**Issue by jmhodges** [bazelbuild/buildifier#37]
I have buildifier as a repo in my WORKSPACE, and was trying to use it with the command

    ./scripts/bazel-run.sh @io_bazel_buildifier//buildifier -- -mode=fix -path=my/path/to/target

(where bazel-run.sh is the same one as in the bazel repo that uses `bazel run --script_path` under the hood)

But that fails to fix up the BUILD in my repo because `-path` is relative the current working directory and, with all `bazel run` commands, the current working directory in in the bazel sandbox, not directory of the source code repository.

It's kind of a bummer to build and run bazel repo'ed commands, so it might be nice to be able to set the current working directory back to the root of the source code repo for these commands.

Maybe this is a request for `bazel run`? Y'all would know better than me.

**Comment by pmbethe09** [comment]
Yes, the running in the sandbox is a deliberate decision by 'bazel run'.
Buildifier wouldn't even know where to cd to if it wanted to do so before running (in this case).

**Comment by jmhodges** [comment]
Right, I'm asking for a param to pass it like gazelle has.
